Content.Shared/Speech/Muting/MutedComponent.cs

```diff
@@ -6,5 +6,11 @@ namespace Content.Shared.Speech.Muting
     public sealed partial class MutedComponent : Component
     {
 
+      /// <summary>
+      /// Whether the entity should be able to be unmuted
+      /// </summary>
+      [DataField]
+      public bool removable = true;
+
     }
```

Comment:
This sucks just use an event instead, also it's not even using C# / SS14 conventions for pascal case.

Reply:
Oops